apps/src/templates/progress/LessonGroup.jsx

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Radium from 'radium';
-import DetailProgressTable from './DetailProgressTable';
-import SummaryProgressTable from './SummaryProgressTable';
-import FontAwesome from '../FontAwesome';
-import {levelType, lessonType} from './progressTypes';
+import {connect} from 'react-redux';
+import DetailProgressTable from '@cdo/apps/templates/progress/DetailProgressTable';
+import SummaryProgressTable from '@cdo/apps/templates/progress/SummaryProgressTable';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
```

Comment:
any particular reason these have changed? Just checking if I'm missing a new best practice since I don't remember any conclusion to the debate of which method of importing things is preferred.

Reply:
Ummm I don't know if anyone else has a rule. I like it better when they are written this way because then if you want to use them somewhere else or move the file it doesn't cause problems.